MapboxNavigation/RouteController.swift

```diff
@@ -130,8 +130,17 @@ extension RouteController: CLLocationManagerDelegate {
             let userHeadingNormalized = wrap(location.course, min: 0, max: 360)
             courseMatchesManeuverFinalHeading = differenceBetweenAngles(finalHeadingNormalized, userHeadingNormalized) <= RouteControllerMaximumAllowedDegreeOffsetForTurnCompletion
         }
-        
-        if userSnapToStepDistanceFromManeuver <= RouteControllerManeuverZoneRadius {
+
+        // When departing, `userSnapToStepDistanceFromManeuver` is most often less than `RouteControllerManeuverZoneRadius`
+        // since the user will most often be at the beginning of the route, in the maneuver zone
+        if alertLevel == .depart && userSnapToStepDistanceFromManeuver >= RouteControllerManeuverZoneRadius {
```

Comment:
This looks like inverted logic to me. The snapped distance from the maneuver has to be _more than_ the maneuver zone radius?